src/CLU/Microsoft.Azure.Commands.Compute/Extension/CustomScript/GetAzureVMCustomScriptExtensionCommand.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.Azure.Commands.Compute
         DefaultParameterSetName = GetCustomScriptExtensionParamSetName),
     OutputType(
         typeof(VirtualMachineCustomScriptExtensionContext))]
-    [CliCommandAlias("vm customscriptextension ls")]
+    [CliCommandAlias("vm extension script ls")]
```

Comment:
because it makes the cmdlet more difficult to type and there is no 'non custom' script extension